src/ControlSystem/ControlErrors/Size/CMakeLists.txt

```diff
@@ -30,15 +26,3 @@ spectre_target_headers(
   Initial.hpp
   RegisterDerivedWithCharm.hpp
   )
-
-target_link_libraries(
-  ${LIBRARY}
-  PUBLIC
-  ApparentHorizons
-  Boost::boost
-  DataStructures
-  ErrorHandling
-  GSL::gsl
-  Interpolation
-  Parallel
```

Comment:
Presumably the library this was merged into needs these now.  (Some are already included.)